Makefile

```diff
@@ -5,6 +5,8 @@
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= 0.0.1
 
+OPERATOR_SDK_VERSION ?= 1.21.0
```

Comment:
why not the latest version `1.22.2`?

Reply:
There are breaking changes which I would like ti understand before upgrading to this version.